pkg/daemon/ceph/osd/volume.go

```diff
@@ -853,6 +853,37 @@ func (a *OsdAgent) appendDeviceClassArg(device *DeviceOsdIDEntry, args []string)
 	return args
 }
 
+func (a *OsdAgent) WipeDevicesFromOtherClusters(context *clusterd.Context) error {
+	for _, device := range a.devices {
```

Comment:
How does this approach only wipe devices that we know we want to provision in the new cluster? Is it because PVC-based clusters only set `a.devices` to a single device? So it would be more complicated for non-pvc OSDs (for a separate PR).